libraries/Microsoft.Bot.Builder/FileTranscriptLogger.cs

```diff
@@ -49,8 +49,8 @@ public FileTranscriptLogger(string folder = null, bool unitTestMode = true)
                 Directory.CreateDirectory(folder);
             }
 
-            this.folder = folder;
-            this.unitTestMode = unitTestMode;
+            this._folder = folder;
```

Comment:
> this._folder = folder; [](start = 11, length = 23)

We should snap to the dotnet core style guidelines [https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md#c-coding-style) 

"An EditorConfig file (.editorconfig) has been provided at the root of the corefx repository, [https://github.com/dotnet/corefx/blob/master/.editorconfig](https://github.com/dotnet/corefx/blob/master/.editorconfig) enabling C# auto-formatting conforming to the above guidelines.

We also use the .NET Codeformatter Tool to ensure the code base maintains a consistent style over time, the tool automatically fixes the code base to conform to the guidelines outlined above. [https://github.com/dotnet/codeformatter](https://github.com/dotnet/codeformatter)"